.golangci.yml

```diff
@@ -134,3 +134,24 @@ linters-settings:
       - typeUnparen
       - unnamedResult
       - unnecessaryBlock
+
+issues:
+  exclude-rules:
+    - linters:
+        - revive
+      text: "if-return"
+    - linters:
+        - revive
+      text: "empty-block"
+    - linters:
+        - revive
+      text: "superfluous-else"
+    - linters:
+        - revive
+      text: "unused-parameter"
+    - linters:
+        - revive
+      text: "unreachable-code"
+    - linters:
+        - revive
+      text: "redefines-builtin-id"
```

Comment:
Excluding `unused-parameter` might be ok for now, but I guess we can still allow other linter rules?

Reply:
Thanks for the review. Current codebase doesn't seems to pass other rules as well. I'll open an following-up PR for refactoring the current code and re-enabling these rules.

Reply:
Following-up: #2371